### PR TITLE
Convert use of `mini_player` in the `video_player_android` test to `AndroidVideoPlayer`.

### PR DESCRIPTION
**Pull request by matanlurey**
Closes https://github.com/flutter/flutter/issues/156424.

Also improved an error message that came up as a result.

### CODE REVIEW

File: packages/video_player/video_player_android/example/integration_test/video_player_test.dart [hunk 1]
```diff
@@ -38,132 +30,121 @@ String getUrlForAssetAsNetworkSource(String assetKey) {
...
 
-      expect(await controller.position, const Duration(seconds: 3));
-    });
+  testWidgets('can seek', (_) async {
+    final int textureId = (await player.create(DataSource(
+      sourceType: DataSourceType.asset,
+      asset: _videoAssetKey,
+    )))!;
 
-    testWidgets('can be paused', (WidgetTester tester) async {
```
Comment (camsim99) [inline packages/video_player/video_player_android/example/integration_test/video_player_test.dart:82]: Should we add back a test for pausing like this one that was removed?

**Review by camsim99 — COMMENTED** [review]
Looks good just a question about a removed test.

**Comment by matanlurey** [comment]
Thanks @camsim99! PTAL again

File: packages/video_player/video_player_android/example/integration_test/video_player_test.dart (hunk 1 above)
Comment (camsim99) [reply packages/video_player/video_player_android/example/integration_test/video_player_test.dart:82]: Looks like there was an additional call to `await tester.pumpAndSettle(_playDuration)` after pause was called in the original test. Maybe that explains the slight difference in `player.getPosition(textureId)` and  `pausedDuration` causing the test failure?

**Review by camsim99 — APPROVED** [review]
LGTM but there's a test failure. Left a comment on maybe why

File: packages/video_player/video_player_android/example/integration_test/video_player_test.dart (hunk 1 above)
Comment (matanlurey) [reply packages/video_player/video_player_android/example/integration_test/video_player_test.dart:82]: Hmm, let's try it :)

**Pull request merged**